presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -2646,6 +2646,16 @@ public void testSimpleLeftJoin()
     {
         assertQuery("SELECT COUNT(*) FROM lineitem LEFT JOIN orders ON lineitem.orderkey = orders.orderkey");
         assertQuery("SELECT COUNT(*) FROM lineitem LEFT OUTER JOIN orders ON lineitem.orderkey = orders.orderkey");
+
+        // With null base in outer side dereference expression in condition
+        assertQuery(
+                "SELECT x.val FROM " +
+                        "(SELECT cast(row(v) as row(val integer)) FROM (values 1,2,3) t(v)) ta(x) " +
```

Comment:
Capitalize `VALUES` and add spaces after comma

Add space between `ta` and `(`